includes/controls/groups/base.php

```diff
@@ -396,6 +396,10 @@ private function init_options() {
 	 */
 	protected function init_args( $args ) {
 		$this->args = array_merge( $this->get_default_args(), $this->get_child_default_args(), $args );
+
+		if ( isset( $this->args['scheme'] ) ) {
```

Comment:
Is it needed? without it it will happen on thr `add_control`, no?

Reply:
Yes, because there are group control methods that process scheme values (They now process globals) (e.g. `start_popover()`)